src/app/qgsmeasuretool.cpp

```diff
@@ -67,6 +69,10 @@ const QList<QgsPoint>& QgsMeasureTool::points()
   return mPoints;
 }
 
+void QgsMeasureTool::openConfigTab()
+{
+  mQgis->showOptionsDialog( mDialog, QString( "mOptionsPageMapTool" ) );
```

Comment:
Thanks for pointing it. I didn't think about it because the only other showOptionsDialog in the QGIS code was into qgscomposer.cpp and it used a pointer to the main QgisApp.

QGisApp instance is much more simple to use and I just need to modify some lines directly into QgsMeasureDialog instead of trying to catch a pointer to QgisApp.